tvision/classes/wingr/wingrscr.cc

```diff
@@ -734,7 +734,12 @@ int TScreenWinGr::System( const char * command
   }
 
   if ( pidChild )                      // Leave in the background
-  { return( *pidChild= (pid_t)pi.hProcess );  // Give process identifier
+  { 
```

Comment:
I don't fully understand this patch. `pidChild` argument is `pid_t *`. What's the idea in casting the value to 64 bits and then store it in a `pid_t`. More in the next patch.